ci/Dockerfile

```diff
@@ -1,9 +1,10 @@
 FROM docker.io/library/debian:stable-slim
 
 ### meta ###
-ENV RUST_STABLE_VERSION=1.85.0 \
+ENV RUST_STABLE_VERSION=1.86.0 \
+    # restricted by https://github.com/trailofbits/dylint/blob/master/examples/general/rust-toolchain
```

Comment:
Based on https://github.com/use-ink/ink/issues/2458#issuecomment-2776631391, makes the reason explicit here.